src/components/FlowRunGraph.vue

```diff
@@ -5,7 +5,9 @@
 </template>
 
 <script lang="ts" setup>
+  import { FlowRunGraphConfig } from '@/models/FlowRunGraph'
+
   defineProps<{
-    flowRunId: string,
+    config: FlowRunGraphConfig,
```

Comment:
This is noteworthy to call out. By accepting the `flowRunId` and a `fetch` method to get the graph data. This component can do its own fetching and refreshing of data. Which means it can fetch its own sub flow data rather than depending on the implementation to listen for events. 

I'm not sure how this will effect the implementation forcing an update. Might expose a `refresh` method on the component instance itself.

Reply:
Ah I dig this! The model binding hurdles were def a bit much there.